apps/web/package.json

```diff
@@ -24,14 +24,15 @@
     "@latitude-data/web-ui": "workspace:*",
     "@lucia-auth/adapter-drizzle": "^1.0.7",
     "@monaco-editor/react": "^4.6.0",
-    "@sentry/nextjs": "^8",
+    "@sentry/nextjs": "^8.32.0",
     "@sentry/utils": "^8.30.0",
     "@sindresorhus/slugify": "^2.2.1",
     "@t3-oss/env-nextjs": "^0.10.1",
     "ai": "^3.2.42",
     "bullmq": "^5.8.5",
     "date-fns": "^3.6.0",
     "drizzle-orm": "^0.33.0",
+    "import-in-the-middle": "1.11.2",
```

Comment:
when removing sentry as a external dependency nextjs prompted me to install this

Reply:
Where is used? Also what does? I read their npm description but I don't get it

Reply:
sentry uses it under the hood